tests/test_backward_compatibility.py

```diff
@@ -67,7 +67,7 @@ def test_stable_register_consumer_args(self):
                                              self.connector.callback_func_1,
                                              self.connector.default_error_handler,
                                              False)
-            self.assertIsInstance(self.connector.consumers['test_consumer'], ConsumerThread)
-            self.assertEqual(self.connector.consumers['test_consumer'].queue, 'test')
+            self.assertIsInstance(self.connector.consumers['test_consumer']['instance'], ConsumerThread)
```

Comment:
This indicates breaking compat.

Reply:
resolved by: https://github.com/NeonGeckoCom/neon_mq_connector/pull/52/commits/a8444413eeedcb11ed2815b392c810a0321ebaca#diff-7821367cfa6709641623fca8f3c50da6c4950b15d8171c8930e60126423e7170R70